core/ui/src/main/java/com/google/samples/apps/nowinandroid/core/ui/NewsResourceCard.kt

```diff
@@ -228,28 +230,40 @@ fun NewsResourceShortDescription(
 
 @Composable
 fun NewsResourceTopics(
-    topics: List<Topic>,
+    topics: List<FollowableTopic>,
     modifier: Modifier = Modifier
 ) {
     // Store the ID of the Topic which has its "following" menu expanded, if any.
     // To avoid UI confusion, only one topic can have an expanded menu at a time.
     var expandedTopicId by remember { mutableStateOf<String?>(null) }
+    val isFollowed = stringResource(R.string.topic_is_followed)
```

Comment:
`isFollowed` implies that this is a boolean, but it's actually text. These variables should be suffixed with `Text` or `ContentDescription`. 

Also, it's best to keep variables as close as possible to where they are used so they can be moved into the `text = { ` block.